ui/perfherder/alerts/TagsModal.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  Button,
+  Form,
+  FormGroup,
+  Label,
+  CustomInput,
+} from 'reactstrap';
+
+import { getData } from '../../helpers/http';
+import { getApiUrl } from '../../helpers/url';
+import { endpoints } from '../constants';
+
+export default class TagsModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tags: [],
+      initialTags: [],
```

Comment:
These two var names are ambiguous. It's not immediately obvious what they're holding.
I actually thought `tags` holds all the available tags, while `initialTags` hold those of the summary.
But after reading the code, I discovered it's actually the other way around.